manifest: Update hal_stm32 module with updated license file

benothmn-st opened the pull request:
Update hal_stm32 module with updated license file

kartben commented:
what's the status of this PR? was the change maybe already picked up as part of another version bump in a different PR?

erwango commented:
>  what's the status of this PR? was the change maybe already picked up as part of another version bump in a different PR?

This PR is still valid. We're actually blocked by internal discussions on the License terms. There were some recent progresses on this front, so I hope we'll be able to resume the PR soon.

benothmn-st committed:
manifest: Update hal_stm32 module with updated license file

Update hal_stm32 module with updated license file.
Impacted source/headers files were updated also.

Signed-off-by: Nidhal BEN OTHMEN <nidhal.benothmen@st.com>
